src/Device/DeviceSetup/DeviceSetupView.tsx

```diff
@@ -37,13 +49,17 @@ export default class DeviceSetupDialog extends React.Component {
             this.props;
         const { selectedChoice } = this.state;
 
-        if (choices && (choices.length > 0 || choices.size > 0)) {
+        if (
+            choices &&
+            (choices.length > 0 ||
+                (choices as unknown as { size: number }).size > 0)
+        ) {
```

Comment:
Can `choices` be either a list of strings or an object with a `size` property?

Reply:
Is this some remnants from `Immutable.js` or something that we can just remove?

Reply:
Good catch! It's actually not exported, so no other apps are using it directly, and internally in shared we are only using string.